pegasus/sites.v3/code.org/public/promote/index.haml

```diff
@@ -23,10 +23,18 @@ social:
   %h1 Promote Computer Science
 
   %div{:id=>'us_content'}
-    #petition-section.section.clear
-      = view :petition
-    :javascript
-      petitionIncluded = true;
+
+    - if request.locale == 'en-US'
+      %div{style: "background-color:#f3f3f3; border-radius:4px; padding:20px; clear:both"}
+        %h3{style: "margin-top: 0px"} Every student in every school should have the opportunity to learn computer science. Join over 100,000 Americans – including CEOs, governors, and education leaders – to ask Congress to fund computer science in grades K-12.
```

Comment:
haha yeah I figured. I had to edit mine too, just so you knew I got what
you were saying. :D

On Thu, May 5, 2016 at 11:23 AM, Brendan Reville <notifications@github.com>
wrote:

> In pegasus/sites.v3/code.org/public/promote/index.haml
> <https://github.com/code-dot-org/code-dot-org/pull/8238#discussion_r62230633>
> :
>
> > @@ -23,10 +23,18 @@ social:
> >    %h1 Promote Computer Science
> >
> >    %div{:id=>'us_content'}
> > -    #petition-section.section.clear
> > -      = view :petition
> > -    :javascript
> > -      petitionIncluded = true;
> > +
> > +    - if request.locale == 'en-US'
> > +      %div{style: "background-color:#f3f3f3; border-radius:4px; padding:20px; clear:both"}
> > +        %h3{style: "margin-top: 0px"} Every student in every school should have the opportunity to learn computer science. Join over 100,000 Americans – including CEOs, governors, and education leaders – to ask Congress to fund computer science in grades K-12.
>
> oh, that's what i typed but i guess it got parsed :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/8238/files/2948f08e2915e5c4294f5d75d53931bf8ba52348#r62230633>
>



-- 
Tanya Parker
Product Manager, Code.org
+1 (562) 318-4137